src/utils/memoFn.ts

```diff
@@ -0,0 +1,79 @@
+// let cachedCalls = 0;
```

Comment:
```suggestion
```

Reply:
Yeah this is used later in the code for debugging so instead of just adding/removing it I left it to help people see the impact